Lib/test/test_types.py

```diff
@@ -831,12 +831,6 @@ def test_union_copy(self):
             self.assertEqual(copied.__args__, orig.__args__)
             self.assertEqual(copied.__parameters__, orig.__parameters__)
 
-    def test_union_parameter_substitution_errors(self):
-        T = typing.TypeVar("T")
-        x = int | T
-        with self.assertRaises(TypeError):
-            x[42]
```

Comment:
Could make this test `x[int, str]` instead.

Reply:
added